disasterinfosite/static/js/app.js

```diff
@@ -97,17 +98,22 @@ $(document).ready(function() {
     : query_lat + "," + query_lng;
   if (!query_lat || !query_lng) location_query_text = "";
   $locationInput.val(location_query_text);
+  $(".info__location").text(location_query_text);
 
   // Set up autocomplete when someone clicks in the input field
   $locationInput.one("click", function() {
     var input = document.getElementById("location-text");
     $locationInput.prop("placeholder", "");
-    placeSearch({
+    var autocomplete = placeSearch({
       key: MAPQUEST_KEY,
       container: input,
       useDeviceLocation: true
     });
     $locationInput.focus();
+
+    // autocomplete.on("change", function(event) {
```

Comment:
I intend to work with this in my next PR - I noticed that the placeSearch results have a lat / lng in them, so there's no need to get that again from MapQuest.

Conversely, it might be easy for me to do reverse geocoding for a map pin drop.